docs/references/backend/user/update-user.mdx

```diff
@@ -1,80 +1,68 @@
 ---
-title: updateUser()
-description: Use Clerk's Backend SDK to update a user.
+title: updateUserMetadata()
```

Comment:
These changes should be reverted for this page -- you ended up converting `updateUser()` to `updateUserMetadata()`

Reply:
aww jeez

Reply:
oh nice catch!!